src/components/fields/TraceSelector.js

```diff
@@ -154,6 +154,9 @@ TraceSelector.propTypes = {
 
 export default connectToContainer(localize(TraceSelector), {
   modifyPlotProps: (props, context) => {
+    if (!context.container) {
```

Comment:
more defensive coding :)

Reply:
that's what was the bug with timeseries?

Reply:
no I don't think so, it was necessary for this PR to work

Reply:
this definitely does not fix the Timeseries issue, just checked, but it does address another crash pathway.